src/openfermion/utils/_lcu_util.py

```diff
@@ -0,0 +1,181 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Utility methods for LCU circuits."""
+
+from __future__ import absolute_import
+from __future__ import division
+
+import math
+
+
+def _partial_sums(vals):
+    total = 0
+    for v in vals:
+        yield total
+        total += v
+    yield total
+
+
+def _differences(weights):
+    p = None
+    have_p = False
+    for w in weights:
+        if have_p:
+            yield w - p
+        p = w
+        have_p = True
+
+
+def _discretize_probability_distribution(unnormalized_probabilities, epsilon):
+    """Approximates probabilities with integers over a common denominator.
+
+    Args:
+        unnormalized_probabilities: A list of non-negative numbers proportional
+            to probabilities from a probability distribution. The numbers may
+            not be normalized (they do not have to add up to 1).
+        epsilon: The absolute error tolerance.
+
+    Returns:
+        A tuple containing:
+            [0]: A list of numerators.
+            [1]: A common denominator.
+
+        It is guaranteed that numerators[i] / denominator is within epsilon of
+        the i'th input probability (after normalization).
+
+        It is guaranteed that the common denominator is a multiple of the
+        number of probabilities in the distribution.
+    """
+    n = len(unnormalized_probabilities)
+    bin_count = math.ceil(1.0 / (epsilon * n)) * n
+
+    cumulative = list(_partial_sums(unnormalized_probabilities))
+    total = cumulative[-1]
+    discretized_cumulative = [int(math.floor(c/total*bin_count + 0.5))
```

Comment:
please have spaces around / and *. Did you run pylint?

Reply:
Done.

No I didn't run pylint. Continuous integration tools are supposed to do that! I tried the linter suggested in the documentation (`pycodestyle`) but it doesn't return any errors for these files regardless of how I do the math expressions...